ps_customtext.php

```diff
@@ -303,12 +303,18 @@ public function getFormValues()
      * @param string|null $hookName
      * @param array $configuration
      *
-     * @return string
+     * @return string|false
      */
     public function renderWidget($hookName = null, array $configuration = [])
     {
         if (!$this->isCached($this->templateFile, $this->getCacheId('ps_customtext'))) {
-            $this->smarty->assign($this->getWidgetVariables($hookName, $configuration));
+            $var_array = $this->getWidgetVariables($hookName, $configuration);
+            if ($var_array === false) {
+
```

Comment:
Uneeded extra line here.

Reply:
@PrestaEdit There is a rule that there must be an extra line above a `return` statement. When does this rule apply and when not.

Reply:
Ok, but this does not answer my question.

Reply:
It's surely answer. No new line between if() statement and return if there is only a return in it.

Reply:
I found it here: https://symfony.com/doc/3.4/contributing/code/standards.html#structure
>Add a blank line before return statements, unless the return is alone inside a statement-group (like an if statement);